Cannot find module 'slash' on install

**Describe the bug**
When running `yarn create strapi-app my-project --quickstart` or `npx create-strapi-app my-project --quickstart` I get the error `Cannot find module 'slash'`, then am unable to run the scaffolded app.

**Steps to reproduce the behavior**
Just run any of the above install scripts in any directory with the following versions listed below.

**Expected behavior**
For the project to be scaffolded without error messages and runnable with `yarn develop`

**Screenshots**
![image](https://user-images.githubusercontent.com/7675752/84846661-46684480-b01d-11ea-92eb-bc9915966edb.png)

**Code snippets**
If applicable, add code samples to help explain your problem.

**System**
- Node.js version: v12.10.0
- NPM version: v6.10.3
- Strapi version: v.3.0.2
- Database: not applicable
- Operating system: macOS Catalina 10.15.5

--- a/packages/core/strapi/lib/services/metrics/admin-user-hash.js
+++ b/packages/core/strapi/lib/services/metrics/admin-user-hash.js
@@ -10,7 +10,7 @@ const crypto = require('crypto');
  */
 const generateAdminUserHash = (strapi) => {
   const ctx = strapi?.requestContext?.get();
-  if (!ctx?.state?.user) {
+  if (!ctx?.state?.user?.email) {
     return '';
   }
   return crypto.createHash('sha256').update(ctx.state.user.email).digest('hex');

